Eatery/Controllers/Eateries/Campus/TabbedPageViewController.swift

```diff
@@ -216,3 +216,13 @@ extension TabbedPageViewController: UIPageViewControllerDelegate {
     }
 
 }
+
+extension TabbedPageViewController: Reloadable {
+    func reload() {
+        self.viewControllers.forEach { controller in
```

Comment:
Delete self. And just make this

```suggestion
        for controller in viewControllers {
```

Simpler and more concise

Reply:
I actually like the `.forEach` better haha but it's your preference